test/system_settings_test.rb

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SystemSettings"
+
+describe "Yast::SystemSettings" do
+  Yast.import "Bootloader"
+
+  subject(:settings) { Yast::SystemSettings }
+
+  before { settings.main }
+
+  describe "#GetPossibleElevatorValues" do
+    it "returns an array with possible schedulers" do
+      expect(settings.GetPossibleElevatorValues).to eq(["cfq", "noop", "deadline"])
```

Comment:
NP: Comparing Arrays is a bit tricky, if the order changes the test will fail. I usually do
```ruby
expect(settings.GetPossibleElevatorValues.sort).to eq(["cfq", "noop", "deadline"].sort)
```